src/main/entities/PasswordGenerator.java

```diff
@@ -1,14 +1,15 @@
-package src.passwordgenerator;
+package entities;
 
-import src.UI.DialogBox;
+import UI.DialogBox;
 
 public class PasswordGenerator {
 
     PasswordGenerator() {
+        //TODO: Move this to UIController
```

Comment:
Just flagging the TODO

Reply:
Yeah these would get removed by Rutvik in his PR.